php_bson.h

```diff
@@ -60,11 +69,7 @@ typedef struct {
 	bool                     is_visiting_array;
 } php_phongo_bson_state;
 
-#if PHP_VERSION_ID >= 70000
-#define PHONGO_BSON_STATE_INITIALIZER  { {{ 0 }}, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL, 0 }
-#else
-#define PHONGO_BSON_STATE_INITIALIZER  { NULL, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL, 0 }
-#endif
+#define PHONGO_BSON_STATE_INITIALIZER  { 0 }
```

Comment:
I don't believe this is right. I had some compilers whine about this change to just have a single { 0 } in Xdebug: https://github.com/xdebug/xdebug/commit/7ad9db738bfa80d69621e4c7f66a04cd361269f9